index.html

```diff
@@ -0,0 +1,230 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./assets/css/styles.css" />
+
+    <title>Jonathan Jefferies Portfolio</title>
+  </head>
+
+<body>
+
+<nav id="navbar" class="nav">
+  <ul class="nav-list">
+    <li>
+      <a href="#aboutme-section">About Me</a>
+    </li>
+    <li>
+      <a href="#projects">Projects</a>
+    </li>
+    <li>
+      <a href="#contact">Contact</a>
+    </li>
+  </ul>
+</nav>
+
+<section id="aboutme-section" class="aboutme-section">
+  <h1>Jonathan Jefferies Portfolio</h1>
+<div class="profile-image">
+  <img src="./assets/images/jonny.jpeg" alt="Jonny and Hannah test">
+</div>
+  <h2>About Me</h2>
+    <p>
+        I've been self taught for 6 months focusing on the basics of HTML, CSS and Python.
+        These basic courses alongside <strong>Freecodecamp.com</strong> and <strong>futurelearn.com</strong> courses;
+        I've recently enrolled in Birmingham University's 6 month intensive Coding Boot Camp.
+        After it finishes I will be a <strong>Stack Web Developer</strong>.
+    </p>
+  <h2>Experience and Qualifications</h2>
+    <ul>
+      <li>Computer Programming for Everyone - University of Leeds Online</li>
+      <li>Learn to Code for the Web - University Leeds Online</li>
+      <li>Creating a Great User Experience for Mobile Apps - University of Leeds Online</li>
+      <li>Software Development Fundamentals: Programming Languages and HTML - University of Leeds Online</li>
+      <li>Software Development Fundamentals: Project Management - University of Leeds Online</li>
+      <li>Software Development Fundamentals: Web Accessibility and CSS - University of Leeds Online</li>
+      <li>HTML Certificate Course - Programming Hub Certificate</li>
+      <li>Python 3 Certificate Course - Programming Hub Certificate</li>
+      <li>CSS Certificate Course - Programming Hub Certificate</li>
+      <li>300hrs Responsive Web Design, Developer Certification - FreeCodeCamp.org online course</li>
+      <li>Full Stack Web Developer - Birmingham University Bootcamp (Sept 21 - March 22)</li>
+      <li>, </li>
```

Comment:
You have a trailing list item here, the comma is being added on a seperate line.

Reply:
deleted extra line